_pages/en_US/homebrew-launcher-(browserhax-2020).txt

```diff
@@ -0,0 +1,63 @@
+---
+title: "Homebrew Launcher (BrowserHax 2020)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+This method of accessing the Homebrew Launcher takes advantage of some browser oversights in the native 3DS Internet Browser.
+
+{: .notice--success}
```

Comment:
I don't really know how the guide notices/boxes work -- what will `{: .notice--success}` display/change? The text above it probably shouldn't be boxed.

Reply:
makes it green
dunno why it was green
i probably copied it from somewhere